spec/jobs/dqt/recommend_for_award_job_spec.rb

```diff
@@ -21,6 +21,20 @@ module Dqt
           described_class.perform_now(trainee)
         }.to change(trainee, :awarded_at).to(Time.zone.parse(award_date))
       end
+
+      it "triggers the trainee update job" do
+        expect(Trainees::Update).to receive(:call).with(trainee: trainee,
+                                                        params: { awarded_at: award_date },
+                                                        update_dqt: false)
+        allow(trainee).to receive(:award!).and_return(true)
```

Comment:
I was getting a test fail here without stubbing the trainee. I think what was happening is, because I was mocking the Trainee::Update, the trainee wasn't actually being updated with the `awarded_at` and was erroring on the line `trainee.award!`

Is my logic here correct? The other two tests where we check the `awarded_at` and the `state` on the trainee are passing.